controllers/post_controller_test.go

```
@@ -0,0 +1,75 @@
+		Published: true,
+	}
+	jsonBody, _ := json.Marshal(body)
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("POST", "/posts", bytes.NewBuffer(jsonBody))
+	req.Header.Set("Content-Type", "application/json")
+
+	r.ServeHTTP(w, req)
+
+	assert.Equal(t, http.StatusCreated, w.Code)
+	var resp models.Post
+	json.Unmarshal(w.Body.Bytes(), &resp)
```

golangci-lint (1.64.8)

Error return value of `json.Unmarshal` is not checked

(errcheck)